playwright-tests/tests/proposal/proposals.spec.js

```diff
@@ -393,21 +450,34 @@ test.describe('Moderator with "Don\'t ask again" enabled', () => {
         return originalResult;
       },
     });
-    await page.goto(`/${account}/widget/app?page=proposal&id=17`);
+    await page.goto(
+      `/${account}/widget/app?page=proposal&id=${
+        isInfrastructureCommittee ? "4" : "17"
+      }`
+    );
 
     const firstStatusBadge = await page
       .locator("div.fw-bold.rounded-2.p-1.px-2")
       .first();
     await expect(firstStatusBadge).toHaveText("REVIEW", { timeout: 10000 });
     await page.locator(".d-flex > div > .bi").click();
 
-    await page.getByTestId("Sponsor verifies KYC/KYB").uncheck();
-    await page.getByRole("button", { name: "Review", exact: true }).click();
-    await page.getByText("Approved", { exact: true }).first().click();
+    const sponsorCheckbox = await page
+      .locator("label")
+      .filter({ hasText: "Sponsor verifies KYC/KYB" })
+      .locator("xpath=preceding-sibling::*[1]");
+    if (isInfrastructureCommittee) {
+      await expect(sponsorCheckbox).toBeDisabled();
```

Comment:
@Megha-Dev-19 Infrastructure committee behaves a bit different here. As you can see it is not possible to uncheck the "Sponsor verifies KYC/KYB" checkbox as in the other instances. Is this the intended behaviour?